utils/README.md

```diff
@@ -63,3 +66,87 @@ To install and configure a system with Kata Containers and containerd, run:
 ```bash
 $ bash -c "$(curl -fsSL https://raw.githubusercontent.com/kata-containers/kata-containers/main/utils/kata-manager.sh)"
 ```
+
+### Choose a Hypervisor
+
+Kata works with different [hypervisors](../docs/hypervisors.md). When you install a Kata system, the default hypervisor
+will be configured, but all the available hypervisors are installed.
+This means you can switch between hypervisors whenever you wish.
+
+#### List available hypervisors
+
+Run the following on an installed system:
+
+```bash
+$ bash -c "$(curl -fsSL https://raw.githubusercontent.com/kata-containers/kata-containers/main/utils/kata-manager.sh) -L"
+```
+
+#### Show the default packaged hypervisor
+
+To show the default packaged hypervisor, run the following on an
+installed system:
+
+```bash
+$ bash -c "$(curl -fsSL https://raw.githubusercontent.com/kata-containers/kata-containers/main/utils/kata-manager.sh) -L" | grep default
+```
+
+#### Show the locally configured hypervisor
+
+`kata-manager.sh` will create a "local" copy of the packaged Kata configuration
+file.
+
+To show details of the _local_ copy of the configuration files on an
+installed system, run:
+
+```bash
+$ bash -c "$(curl -fsSL https://raw.githubusercontent.com/kata-containers/kata-containers/main/utils/kata-manager.sh) -e"
+```
+
+> **Note:** This command can only be run once Kata has been installed.
+
+> See the [configuration documentation](https://github.com/kata-containers/kata-containers#configuration)
+> for further information.
```

Comment:
I wonder if those 3 options could be combined into a single one which would output something like:
```
HYPERVISOR | ARGUMENT NAME | DEFAULT
Cloud Hypervisor | clh | yes
QEMU | qemu | no
,,,,
```
Or something towards that line ...
NOTE: After the next release e should remove all the mentions to `$ bash -c ...` and just point to the script directly.

Reply:
> I wonder if those 3 options could be combined into a single one

I'm not totally clear what you mean by this comment but I have updated the hypervisors doc to include more hypervisor config details including the "short name" used by `kata-manager`.

> NOTE: After the next release e should remove all the mentions to $ bash -c ... and just point to the script directly.

Yep - see #9091.